docs/zh-cn/open-api.md

```diff
@@ -257,10 +257,7 @@ POST
 * 请求示例
 
 ```
-http:serverIp:8848/nacos/v1/cs/configs
-
-http body：
-dataId=dataIdparam&group=groupParam&tenant=tenantParam&content=contentParam&type=typeParam
+curl -X POST 'http://localhost:8848/nacos/v1/cs/configs?dataId=nacos.example&group=com.alibaba.nacos&content=contentTest'
```

Comment:
Post should use body? the old example seems that means

like `curl -X POST 'http://localhost:8848/nacos/v1/cs/configs' -d 'dataId=nacos.example&group=com.alibaba.nacos&content=contentTest'